tee-worker/identity/client-sdk/packages/client-sdk/src/lib/type-creators/tc-authentication.ts

```diff
@@ -21,9 +25,11 @@ export function createTCAuthenticationType(
     [data.type]:
       data.type === 'Email'
         ? data.verificationCode
-        : createLitentryMultiSignature(registry, {
+        : data.type === 'Web3'
```

Comment:
How about changing it to map or switch case? It might be more intuitive. I think there might be additional types added here in the future.

for example:
```
const getValue = () => {
  switch (data.type) {
    case 'Email':
      return data.verificationCode;
    case 'Web3':
      return createLitentryMultiSignature(registry, {
        who: data.signer,
        signature: data.signature,
      });
    default:
      return data.token;
  }
};

[data.type]: getValue()
```